### ISSUE TITLE
[0.9.1.9 beta release-136] USER ISSUE: "Sever encountered an exception" message

### ISSUE THREAD

**Issue by MKVangen** [StrangeLoopGames/EcoIssues#20068]
\
**Version:** 0.9.1.9 beta release-136

**Steps to Reproduce:**
I don't know. It just popped up while walking

**Expected behavior:**

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: InvalidOperationException
Message:Collection was modified; enumeration operation may not execute.
Source:System.Private.CoreLib

System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
   at System.Collections.Generic.List`1.Enumerator.MoveNextRare()
   at Eco.Shared.Utils.EnumerableExtensions.ToBson(IEnumerable enumeration, INetClient client, Boolean doAllParams)
   at Eco.Shared.Serialization.BsonManipulator.ToBson(Object val, INetClient client, Boolean useReflection)
   at Eco.Core.Controller.ControllerManager.PackageValue(IMvcNetClient boundClient, Object value, Int32 nameID)
   at Eco.Core.Controller.ControllerManager.PackageProperty(IMvcNetClient boundClient, IController controller, ISyncPropertyInfo propertyInfo)
Outer Exceptions:
Error during packaging property 'PlayerCountAtEachLevel' of 'Eco.Gameplay.Skills.SkillTree'</size>
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No mods installed

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by MKVangen** [comment]
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/5832423/Player-prev.log)
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/5832424/Player.log)

**Comment by SlayksWood** [comment]
THis is fixed in 0.9.2
